### ISSUE TITLE
Kanban view attempting to parse python fragments instead of JS. 

### ISSUE THREAD

**Issue by ayo-odoo** [trinhanhngoc/pycharm-odoo#46]
![image](https://user-images.githubusercontent.com/75064494/141668452-756b4675-2cf7-44df-8fa2-8ab32a120bbe.png)

**Comment by trinhanhngoc** [comment]
@ayo-odoo Could you please give me a complete code example to reproduce this issue?

**Comment by ayo-odoo** [comment]
@trinhanhngoc 

```xml
<record id="stock_picking_type_kanban" model="ir.ui.view">
    <field name="model">stock.picking.type</field>
    <field name="inherit_id" ref="stock.stock_picking_type_kanban"/>
    <field name="arch" type="xml">
        <xpath expr="//templates" position="before">
            <field name="should_show_receipt_kanban_button"/>
            <field name="is_current_user_driver"/>
            <field name="is_picking_type_receipt"/>
        </xpath>

        <xpath expr="//templates//div[hasclass('o_kanban_card_content')]" position="attributes">
            <attribute name="t-if">!(is_current_user_driver and is_picking_type_receipt)</attribute>
        </xpath>
    </field>
</record>
```

The attribute `<attribute name="t-if">!(is_current_user_driver and is_picking_type_receipt)</attribute>` should be parsed as a javascript fragment. It is getting parsed as a python instead. 

Thank you so much for the great work

**Comment by trinhanhngoc** [comment]
@ayo-odoo Thank you for your report. This issue has just been fixed and will available in the next version 2021.2.11.
